src/components/images/style.css

```diff
@@ -12,12 +12,22 @@
     opacity: 0;
 }
 
+@keyframes fadein {
+    from {
+        opacity: 0;
+    }
+
+    to {
+        opacity: 1;
+    }
+}
+
 .lazy-blurhash-fadein-fast {
-    transition: opacity 0.2s;
+    animation: fadein 0.1s;
 }
 
 .lazy-blurhash-fadein {
-    transition: opacity 0.7s;
+    animation: fadein 0.4s;
```

Comment:
Why those values? Why not 0.2 and 0.7?

Imo, we should stick to an specific speed for each mode. In #1458 you're still using the old values of 0.7 and 0.2. Either those or 0.1 or 0.4, but not both combined crazily

Reply:
The normal ones feel too long, since the images on top also fades-in.

Animations in different places can have different length, we're not forced to stick to one all over the place. They ideally need to be adjusted individually for both settings, which is what was done here.

Reply:
@MrTimscampi Fair enough